js/app.js

```diff
@@ -0,0 +1,22 @@
+$(function() {
+
+  $("#stopButton").click(turnRed);
+  $("#slowButton").click(turnYellow);
+  $("#goButton").click(turnGreen);
+
+  function turnRed() {
+    $(".bulb").css("background-color", "black");
```

Comment:
@aaronbatsss this could have been made into a function, and the other functions could have called it.